Examples/SmokeTest/SmokeTestUITests/SmokeTestUITests.swift

```diff
@@ -153,6 +153,17 @@ final class SmokeTestUITests: XCTestCase {
         app.buttons["Flush"].tap()
     }
 
+    func testUIKitInstrimentation() throws {
+        let app = XCUIApplication()
+        app.launch()
+        app.buttons["UIKit"].tap()
+        XCTAssert(app.staticTexts["Sample UIKit App"].waitForExistence(timeout: uiUpdateTimeout))
```

Comment:
Since we're swizzling `UIViewController`, this instrumentation should work for both `UINavigationContoller` and `UITabBarController`, among others. Should we add tests for both a navigation controller and a tab view, to make sure they're both reasonable? I'm not sure how annoying that would be to build in Storyboard.